src/styles/blockes/card.scss

```diff
@@ -0,0 +1,151 @@
+
+html {
+  font-family: Roboto, sans-serif;
+  color: $main-accent;
+}
+
+p {
+  margin: 0;
+}
+
+.prod-card-container {
+justify-content: center;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .prod-card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .prod-card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .prod-card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+@media (min-width:1200px) {
+  .prod-card-container {
+    padding: 50px 128px;
+  }
+}
```

Comment:
It`s not a part of the `product-card` block, so please move it to a separate file.